src/main/java/billgates/entities/UserFactory.java

```diff
@@ -0,0 +1,30 @@
+package billgates.entities;
+
+public class UserFactory {
+    //use getShape method to get object of type shape
+    public IUser getUser(int id, String name, String UserType) {
```

Comment:
I believe the factory method doesn't work like this? 

We should have a way to store the information, then call multiple generation methods to return the User types we are interested in. 
E.g.
```
private int id = -1;
private String name = "";
private String password = "";
private int billId = -1;

public void setId(int id) { this.id = id; }
public void setName(String name) { this.name = name; }
public void setPassword(String password) { this.password = password; }

public DummyUser generateDummyUser() { return DummyUser(id, name); }
public User generateUser() { return User(id, name, password, billId); }
```